example/lib/star_wars/planet_list.dart

```diff
@@ -0,0 +1,150 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_hooks/flutter_hooks.dart';
+import 'package:flutter_hooks_gallery/star_wars/hooks.dart';
+import 'package:flutter_hooks_gallery/star_wars/redux.dart';
+import 'package:flutter_hooks_gallery/star_wars/star_wars_api.dart';
+import 'package:provider/provider.dart';
+
+/// This example will load, show and let you navigate through all star wars
+/// planets.
+///
+/// It will demonstrate on how to use [Provider] and redux ([useReducer])
+class PlanetList extends HookWidget {
+  @override
+  Widget build(BuildContext context) {
+    /// create single instance of Star Wars Api
+    final api = useMemoized(() => StarWarsApi());
+
+    /// create single instance of redux store
+    final store = useReducer(
+      reducer,
+      initialState: AppState(),
+    );
+
+    return Scaffold(
+      appBar: AppBar(
+        title: const Text(
+          'Star Wars Planets',
+        ),
+      ),
+
+      /// provide star wars api instance,
+      /// redux store instance
+      /// and redux state down the widget tree.
+      body: MultiProvider(
+        providers: [
+          Provider.value(value: api),
+          Provider.value(value: store),
+          Provider.value(value: store.state),
+        ],
+        child: HookBuilder(
```

Comment:
Why such `HookBuilder` when `state` and `store` are available from PlanetList directly?